### ISSUE TITLE
BranchSummary regex only works when `-v` is passed as an option to git.branch

### ISSUE THREAD

**Issue by Whoaa512** [steveukx/git-js#321]
The following results in an empty array even though the output of the git command says otherwise:

```javascript
(async () => {
    var git = SimpleGit(process.cwd())
    var result = await git.branch(['--list', '--remote'])

    console.log(result.all)
    // => []
})()
```

**Comment by Whoaa512** [comment]
Looking further, it seems that if you the branch summary regex only works when the `-v` flag is passed. 

Will update the issue name

**Comment by steveukx** [comment]
Hi, thanks for spotting this. A fixed version is now available as `1.106.0` on npm.
